src/components/atoms/Button/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import ButtonIcon from '../Button-icon'
+
+import styles from './styles.css'
+
+const Button = ({ onClick, text }) => (
+  <button className={styles.button} onClick={onClick}>
```

Comment:
We might add ``classNames lib`` so with this the component become more generic to other projects because we can change the colour according to the style guide of each project